src/js/views/Settings.js

```diff
@@ -205,14 +203,19 @@ class Settings extends React.Component {
             </div>
           </label>
 
-          <Route path="/settings/:server?/:id?" component={Servers} />
-
+          <Routes>
+            <Route path=":server/*" element={<Servers />} />
+          </Routes>
           <h4 className="underline">
             <I18n path="settings.services.title" />
             <a name="services" />
           </h4>
-
-          <Route path="/settings/:services?/:service?/:id?" component={Services} />
+          <Routes>
+            <Route path="" element={<Services />} />
+            <Route path=":services/" element={<Services />} />
+            <Route path=":services/:service/" element={<Services />} />
+            <Route path=":services/:service/:id" element={<Services />} />

```

Comment:
If all sub-paths resolve to `<Services />` couldn't we just use a wildcard path?